Examples/Image/FeatureExtraction/FeatureExtraction.py

```diff
@@ -34,7 +34,7 @@ def eval_and_write(model_file, node_name, output_file, minibatch_source, num_obj
             output = output_nodes.eval(mb[features_si])
 
             # write results to file
-            out_values = output[0,0].flatten()
+            out_values = output[0][0].flatten()
```

Comment:
The output of output_nodes.eval() has changed shape from an [array([[[[,]]]])] into an array([[[[[,]]]]]). The latter does not support multi-dimensional indexing. I'm not sure what the source of this indexing change was (or if it was intended). It would be good to verify that a switch to [][] indexing is a correct replacement in general (i.e. the change didn't also transpose the data).